docs/use_cases/model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratosphereQBO.py

```diff
@@ -0,0 +1,158 @@
+# the zonal and meridional mean at 30 and 50mb are output as time series, and the bias (ME) 
+# is computed.  The 30 and 50mb U time series are also plotted.
+#
+
+##############################################################################
+# Datasets
+# --------
+#
+# GFS 0-24 hour forecasts for 10/2017 - 2/2018
+# ERA: 30 year climatology for EOFs and 10/2017 - 2/2018  
+#
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case runs the UserScript wrapper tool to run a user provided script,
+# in this case, stratosphere_qbo_driver.py to compute the zonal and meridional
+# mean, compute anomalies, create EOFs, create Phase diagrams, output U at 30 and
+# 50mb, and create time series of 30 and 50mb winds.  StatAnalysis is then run on 
+# the 30 and 50mb winds to compute the bias (ME).
+#
+
+##############################################################################
+# METplus Workflow
+# ----------------
+#
+# This use case does not loop but plots the entire time period of data.  The 
+# following tools are run once: UserScript, StatAnalysis
+# 
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config,
+# then it loads any configuration files passed to METplus via the command line
+# with the -c option, i.e. -c parm/use_cases/model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratosphereQBO.conf
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratosphereQBO.conf
+#
+
+#############################################################################
+# MET Configuration
+# ---------------------
+#
+# METplus sets environment variables based on user settings in the METplus configuration file. 
+# See :ref:`How METplus controls MET config file settings<metplus-control-met>` for more details. 
+#
+# **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!**
+#
+# If there is a setting in the MET configuration file that is currently not supported by METplus you'd like to control, please refer to:
+# :ref:`Overriding Unsupported MET config file settings<met-config-overrides>`
+#
+# **StatAnalysisConfig_wrapped**
+#
+# .. note:: See the :ref:`Stat-Analysis MET Configuration<series-analysis-met-conf>` section of the User's Guide for more information on the environment variables used in the file below:
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/met_config/STATAnalysisConfig_wrapped
+#
+
+##############################################################################
+# Python Embedding
+# ----------------
+#
+# This use case does not use python embedding.
+#
+
```

Comment:
We are trying to add a new section in the documentation to capture those use cases that use Python to perform calculations outside of METplus. The "Python Scripting" section should follow something close to the Python Embedding section, discussing what is being done by the script and why the decision was made to use Python scripting rather than Python embedding. Here's a link to the template for further reference: https://docs.google.com/document/d/1lhGU9cI3uNmioQBkqBhk7nb-USpljee8Q5F6nzUPTFA/edit?usp=sharing

Reply:
How is this python script section different from what is in the METplus components section?   It seems like the same information would need to be in both?

Reply:
Good question; METplus Components is only to state what tools are being used. In this case, UserScript and StatAnalysis (I think you should also list the Analysis tools, as you correctly did). You do not state *what* those tools are doing in that section, only that you are using them. In the METplus Workflow section, you should discuss what each METplus tool is doing (very briefly, no need to be in-depth). But only those tools within METplus; the discussion of what the Python script does is saved for the Python Scripting section.

The document I linked to provides a brief description of each of the sections, as well as an example of what might be in one. We tried to avoid duplication of information, but that meant we had to rearrange where information is located.

Reply:
Okay, I think the problem here is that since I am using multiple repositories (which are also METplus components), it naturally makes sense to describe what the driver script does here so I can link that with the multiple repositories, etc.  I think it may be confusing to the user to break up all of those discussions so I'm hesitant to do that.  I put in a python scripting section and just stated to see the METplus components section for a discussion of what it does.  I hope that's an applicable solution.

Reply:
Since your use case would be one of the first to utilize the Python Scripting section and we are short on time for this getting in the repo, let's go with this for now. I'll make a personal note to review this in conjunction with the use case template with the METplus wrappers group in an upcoming meeting for feedback and see if the template needs to be revised, what should change with this use case's documentation, etc.

Reply:
Sounds good.  Multi-repository driver scripts are a different animal than more "typical" METplus use cases, so it might even make sense to have a slightly different template for them, as compared to more simple METplus use cases